playwright/tests/extension.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { test as base, expect, BrowserContext, chromium } from '@playwright/test'
+import path from 'path'
+
+import extensionManifest from '../../build-dev/manifest.json'
+const pathToExtension = path.join(__dirname, '../../build-dev')
```

Comment:
Shouldn't we test prod build in e2e?

Reply:
👍  agree

Maybe only on CI tho? So it is easier to run locally, and rerun on changes

And same for web